modules/math/src/main/java/com/opengamma/strata/math/MathUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2019 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.math;
+
+/**
+ * Simple utilities for maths.
+ */
+public final class MathUtils {
+
+  /**
+   * Returns the power of 2 (square).
+   * 
+   * @param value  the value to check
+   * @return {@code value * value}
+   */
+  public static double pow2(double value) {
+    return value * value;
+  }
+
+  /**
+   * Returns the power of 3 (cube).
+   * 
+   * @param value  the value to check
+   * @return {@code value * value * value}
+   */
+  public static double pow3(double value) {
+    return value * value * value;
+  }
+
+  /**
+   * Returns the power of 4.
+   * 
+   * @param value  the value to check
+   * @return {@code value * value * value * value}
+   */
+  public static double pow4(double value) {
+    return value * value * value * value;
```

Comment:
Since we're doing micro optimisation, is there any performance difference between this and doing 
```
double sq = value * value;
return sq * sq;
```
Or is that equivalent to what's done in hotspot?